src/main/java/org/joda/time/tz/src/northamerica

```diff
@@ -2589,7 +2644,7 @@ Zone America/Dawson	-9:17:40 -	LMT	1900 Aug 20
 # http://puentelibre.mx/noticia/ciudad_juarez_cambio_horario_noviembre_2022/
 
 # Rule	NAME	FROM	TO	-	IN	ON	AT	SAVE	LETTER/S
-Rule	Mexico	1931	only	-	May	1	23:00	1:00	D
+Rule	Mexico	1931	only	-	April	30	0:00	1:00	D
```

Comment:
Please note that this is wrong and has been fixed in upstream https://github.com/eggert/tz/commit/926b507fa5c3192b1b68fab5910cbd3ba9377c97

Reply:
It isn't wrong. The file format allows `April`. Joda-Time now parses it.

Reply:
That's good to know. Thank you. There are other parsers not supporting this, so just wanted to ask.